### PR TITLE
feat: wishlist page added

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
Wishlist management page added and also links updated for each page

### CODE REVIEW

**Review by mariyasada — APPROVED** [review]
looks good to me

File: assets/wishlist-page.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        <section class="main-section">
+            <div class="prod-heading upper-text">My Wishlist</div>
+            <div class="wishlist-container">
+                <div class="card">
+                    <div class="container">
+                        <div class="image">
+                            <img src="/assets/home/image-4.jpg" alt="sunset-and-glass-mug" class="img">
+                            <div class="img-badge">
+                                <i class="fa fa-heart"></i>
+                            </div>
```
Comment (JantuDeb) [inline assets/wishlist-page.html:46]: You can make the badge circular by applying the bellow styles.
` .img-badge{`
    `position: absolute;`
    `padding: 0.5rem;`
    `top: 0.2rem;`
    `right: 0.2rem;`
    `align-items: center;`
    `justify-content: center;`
   `display: flex;`
    `width: 1rem;`
    `height: 1rem;`
`}`

**Review by JantuDeb — APPROVED** [review]
Everything looks perfectly fine, added a suggestion

File: assets/wishlist-page.html (hunk 1 above)
Comment (Kothalkarpriya) [reply assets/wishlist-page.html:46]: Great, I will definitely try it

**Pull request merged**